### ISSUE TITLE
Dedicated Server crash when placing bellows

### ISSUE THREAD

**Issue by Kurtchekov** [codetaylor/pyrotech#15]
When placing down a bellows, the server drops the client connection while throwing the following error:

```
java.util.concurrent.ExecutionException: java.lang.NoSuchMethodError: net.minecraft.util.math.MathHelper.func_180185_a(FF)Z
Caused by: java.lang.NoSuchMethodError: net.minecraft.util.math.MathHelper.func_180185_a(FF)Z
        at com.codetaylor.mc.pyrotech.modules.tech.machine.block.BlockBellows.func_185496_a(BlockBellows.java:46) ~[BlockBellows.class:?]
```

It seems to be about Math.EpsilonEquals function call.

#### Affected Versions

* Minecraft: 1.12.2
* Forge: 14.23.5.2838
* Dropt: 1.14.0
* Pyrotech: 1.0.2
* Athenaeum: 1.17.1

**Comment by codetaylor** [comment]
![image](https://user-images.githubusercontent.com/1240716/59524274-41ac0600-8e88-11e9-924c-ed9e7a8fbdc6.png)

Yeah, looks like it's client-side only... but... why!?! LOL

**Comment by Kurtchekov** [comment]
And here I was, thinking that Mathematics was an "universal language"... =P

**Comment by codetaylor** [comment]
Fixed in 1.12.2-1.0.3
